components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.cpp

```diff
@@ -381,6 +396,19 @@ void Nudging::run_impl (const double dt)
   // Perform horizontal remap (if needed)
   m_horiz_remapper->remap(true);
 
+  // bypass copy_and_pad and vert_interp for skip_vert_interpolation:
+  if (m_skip_vert_interpolation) {
+    for (const auto& name : m_fields_nudge) {
+      auto tmp_state_field = get_helper_field(name+"_tmp");
```

Comment:
The field with `_tmp` suffix is not created during init if `m_skip_vert_interpolation=true`...

Reply:
the !m_skip_vert_interpolation is only for *p_mid* fields right (updated L281-306)

Reply:
the tmp fields for others (T_mid, qv, horiz_winds) are set here for all cases:
```
    // Second copy of the field: after horiz interp (alias "ext" if no remap)
    Field field_tmp;
    if (m_refine_remap) {
      field_tmp = create_helper_field(name_tmp, layout_tmp, grid_tmp->name());
    } else {
      field_tmp = field_ext.alias(name_tmp);
      m_helper_fields[name_tmp] = field_tmp;
    }
```

Reply:
Ah right.